test/powershell/Modules/Microsoft.PowerShell.Management/Resolve-Path.Tests.ps1

```diff
@@ -23,4 +23,7 @@ Describe "Resolve-Path returns proper path" -Tag "CI" {
         $result = Resolve-Path -LiteralPath "TestDrive:\\\\\"
         ($result.Path.TrimEnd('/\')) | Should Be "TestDrive:"
     }
+    It "Resolve-Path -Relative should return correct path on different drive" -Skip:(!$IsWindows) {
+        Resolve-Path -Path "HKCU:\Software" -Relative | Should Be "HKCU:\Software"
+    }
```

Comment:
The code change is not windows specific. The tests should not be windows specific either. I think this can be tested using `Env:` provider on all platforms.

Reply:
The reason I did not use `Env:` is `Resolve-Path -Relative` would strip the `Env:\` part, I am not sure whether this is a bug or not.

Can we do things like
```PowerShell
New-PSDrive -Name PSHomeDrive -PSProvider FileSystem -Root $PSHOME
Resolve-Path -Path "PSHomeDrive:\Modules" -Relative | Should Be "PSHomeDrive:\Modules"
```
instead?

Reply:
Sure, that would work as well. Just instead of $PSHome use $TestDrive.